01.12.2025/binaryCompare.c

```diff
@@ -0,0 +1,68 @@
+#include <limits.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+#define ARRAY_SIZE(arr) (sizeof(arr) / sizeof(arr[0]))
+
+/*
+ * Compare two unsigned numbers.
+ * Return 0, if a == b,
+ * Return 1, if a > b,
+ * Return -1, if a < b.
+ */
+int binaryCompare(unsigned int a, unsigned int b)
```

Comment:
```suggestion
int binaryCompare(unsigned a, unsigned b)
```
Вообще по стайлгайду так, но на контрольной приемлимо